static/css/fonts.css

```
@@ -6,18 +6,19 @@
        url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/SourceHanSerifCN-Regular-yihui.woff2') format('woff2');
 }
 @font-face {
-  font-family: 'FandolKai';
+  font-family: 'Kai';
   src: local('Kaiti'), local('Kaiti SC'), local('STKaiti'), local('楷体'), local('SimKai'), local('AR PL KaitiM GB'), local('DFKai-SB'), local('FandolKai'),
-       url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/AR-PL-KaitiM-GB-yihui.woff2') format('woff2');
+       url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/gkai00mp-yihui.woff2') format('woff2'),
+       url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/bkai00mp-yihui.woff2') format('woff2');
```

既然你已经研究到这一步了，那么再交给你一个小任务，就是看看繁体的文鼎楷体里是不是其实包含了简体字。我看繁体的子集只有 934Kb，简体有 1.17Mb。如果繁体字体里已经包含了简体字，那可能就没必要加载或使用简体的子集了。

```suggestion
       url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/bkai00mp-yihui.woff2') format('woff2'),
       url('https://cdn.jsdelivr.net/gh/yihui/cron/fonts/gkai00mp-yihui.woff2') format('woff2');
```

---

好的，这事交给我。

---

[文鼎简体和繁体测试.pdf](https://github.com/user-attachments/files/15889142/default.pdf)

以你博客所有用过的字，加上我博客需要用到的 5 个字（嫣甝萼竺谕），一共 4645 个字作为样本，安装这两种字体后，在 MS Word 里简单测试了下，繁体对简体的支持很不理想。所以简体放前面是对的。

题外话，测试的方法是不是很不优雅？我记得你在博客里介绍过一种方法就是通过测量字符的长度来判断用的是那种字体，这样就可以精确的统计下繁体对这 4645 个字支持度。

---

乗啰喆嗻嘚変嫑寛沢涙祎糵螀説跶鴂，这几个字文鼎简体还是繁体依然不能正常渲染。某一天或许还得增加其它的楷体进来。

---

> 我记得你在博客里介绍过一种方法就是通过测量字符的长度来判断用的是那种字体，这样就可以精确的统计下繁体对这 4645 个字支持度

这办法对中文应该没有用，中文字符宽度都是一样的。

---

把所有的一万多字放在第一列，设置为字体最全的思源宋体，第一列复制到第二列，然后设置为文鼎楷体，转为图像，通过机器学习的办法把两列长得一摸一样的找出来，然后专门针对这些缺的字，找到文鼎楷体的补集，还有其它开源楷体我还没试过。